test/Feature/RootSignatures/StaticSamplers.test

```diff
@@ -0,0 +1,212 @@
+#--- source.hlsl
+
+Texture2D<float4> In : register(t0);
+
+SamplerState DefaultSampler: register(s0);
+SamplerState OtherSampler: register(s1);
+
+RWTexture2D<float4> DefaultOut : register(u1);
```

Comment:
nit: naming with 'default' and 'other' seems a little weird to me. Maybe something like 'Sampler1' and 'Sampler2' ?